locales/en/app.ftl

```diff
@@ -798,3 +798,23 @@ vpn-promo-cta = Get { -brand-mozilla-vpn }
 
 vpn-promo-headline-new = Save 50% with a full year subscription
 vpn-promo-copy-new = Protect your online data—and choose a VPN subscription plan that works for you.
+
+## VPN educational/promotional banner.  HTML tags should not be translated, e.g. `<em>`
+# user's location is populated dynamically by 3rd-party service, eg: "Your location: Los Angeles, CA".  The 3rd-party service will need to provide localization.
+vpn-banner-location = Your location:
```

Comment:
It is possible to expose the actual localization as a placeholder in the string? This basically comes down to [avoid hard-coding characters](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#dont-hardcode-characters).

Same applies to `vpn-banner-ip-address`.

Reply:
done for both `vpn-banner-location` and `vpn-banner-ip-address`